addons/binding/org.openhab.binding.homematic/README.md

```diff
@@ -163,7 +171,7 @@ All channels have two configs:
 The receiveDelay is handy for dimmers and rollershutters for example. If you have a slider in a UI and you move this slider to a new position, it jumps around because the gateway sends multiple events with different positions until the final has been reached. If you set the ```receiveDelay``` to some seconds, these events are filtered out and only the last position is distributed to openHab. The disadvantage is of course, that all events for this channel are delayed. 
 
 ```
-  Thing HM-LC-Dim1T-Pl-2    JEQ0999999 "Name"  @  "Location" {
+  Thing HM-LC-Dim1T-Pl-2 MyDevice "Name"  @  "Location" [address="JEQ0999999"] {
```

Comment:
Spaces are slightly different between above and here. Might I suggest one space around the `@`?